assets/python-thunar/internxt-virtual-drive.py

```
@@ -0,0 +1,94 @@
+    def _copy_to_clipboard(self, value):
+        clipboard = Gtk.Clipboard.get_default(Gdk.Display.get_default())
+        clipboard.set_text(value, -1)
+        clipboard.store()
+
+    def _copy_internxt_link(self, menu_item, file_info):
+        base64_encoded = self._encode_file_path(file_info)
+        url = base_url + 'copy-link/' + base64_encoded
+
+        request = urllib.request.Request(url, method='POST')
+        try:
+            with urllib.request.urlopen(request, timeout=REQUEST_TIMEOUT_SECONDS) as response:
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)